backend/src/common/models/workspace/workspaceResponse.ts

```diff
@@ -0,0 +1,5 @@
+export interface IWorkspaceResponse {
+  workspaceId: string;
+  name: string;
+  createdByUserId: string;
+}
```

Comment:
file name should be the same as interface 'I...'
do the same for all

Reply:
https://github.com/BinaryStudioAcademy/bsa-2020-chatito/pull/14/commits/d60a0ac7a61cf41a25aa888936b806125a7cf3ef